docs/CHANGELOG.md

```diff
@@ -17,8 +17,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- Reports can be exported and imported via API. Partially fixes [1693](https://github.com/ICTU/quality-time/issues/1693).
 - When measuring security warnings with OWASP ZAP as source, allow for counting alert types as security warnings as opposed to alert instances. Closes [#1902](https://github.com/ICTU/quality-time/issues/1902).
-- Reports can be exported and imported via API. Partially fixes [1693](https://github.com/ICTU/quality-time/issues/1693)
+- Added support for Axe-core JSON files (or zips with Axe-core JSON files) as source for measuring accessibility violations. Closes [#1981](https://github.com/ICTU/quality-time/issues/1981).
+
+### Removed
+
+- Since the JSON format produced by axe-selenium-python is basically the same as the format produced by Axe-core, sources of type axe-selenium-python in existing reports are automatically changed into Axe-core. No user action is needed. It is no longer possible to select axe-selenium-python as source type for accessibility violations.
```

Comment:
Is it worth it to mention in the user docs that axe-selenium-python is still supported by the Axe-core source needs to be used for that?

Reply:
Yes, good point.